website/docs/usage/models.md

```diff
@@ -515,8 +532,8 @@ Because pipeline packages are valid Python packages, you can add them to your
 application's `requirements.txt`. If you're running your own internal PyPi
 installation, you can upload the pipeline packages there. pip's
 [requirements file format](https://pip.pypa.io/en/latest/reference/pip_install/#requirements-file-format)
```

Comment:
This link should be updated: https://pip.pypa.io/en/latest/reference/requirements-file-format/

And I think I'd switch to the `package @ URL` format with the wheel URL in the example below because most people are using wheels and the format is easier to parse. (I'd make a suggestion but it's out the range.)

Reply:
OK, updated. I also removed the paragraph about the `#egg=` fragment since it's no longer in the example.